packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -102,10 +105,10 @@ const TradersHub = observer(() => {
                         ) : (
                             <ButtonToggleLoader />
                         )}
-                        {renderOrderedPlatformSections(
-                            selected_platform_type === 'cfd',
-                            selected_platform_type === 'options'
-                        )}
+                        <OrderedPlatformSections
+                            is_cfd_visible={selected_platform_type === 'cfd'}
+                            is_options_and_multipliers_visible={selected_platform_type === 'options'}
+                        />
```

Comment:
this is not related to my card. just changed the way we are rendering the component.